app/views/users/login_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<div style="text-align: center;">
+  <h2>Log In</h2>
+  <%= form_with url: "/log_in", method: :post, data: { turbo: false } do |form| %>
+    <%= form.label :email, "Email: " %>
+    <%= form.text_field :email %><br>
+
+    <%= form.label :password, "Password: " %>
+    <%= form.text_field :password %><br>
+    
+    <%= form.submit "Log In" %>
+  <% end %>
+  <br> 
+
+  <%= image_tag("logo.jpeg", style: "width: 400px; height: auto;")%>
+</div>
```

Comment:
Later on it would be great to put the logo in the..."header view?" (the name is escaping me right now), so it stays in the same place on these first few screens.